### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mayureshwaykole**
updated SR latch example code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,18 +21,32 @@ Here's an example of SR latch constructed from a pair of cross-coupled NOR gates
 ![SR latch | Source: Wikipedia](https://upload.wikimedia.org/wikipedia/commons/c/c6/R-S_mk2.gif)
 
 ```python
+
 from BinPy import *
 
-a = Connector()
-b = Connector()
+NOR1 = Nor('NOR1')  #First NOR gate
```
Comment (rajat974) [inline README.md:27]: Wrong ! Its NOR and not Nor()

File: README.md [hunk 2]
```diff
@@ -21,18 +21,32 @@ Here's an example of SR latch constructed from a pair of cross-coupled NOR gates
...
+NOR2 = Nor('NOR2')  #Second NOR gate
```
Comment (rajat974) [inline README.md:28]: it doesn't take 'NOR2' as an input

File: README.md [hunk 3]
```diff
@@ -21,18 +21,32 @@ Here's an example of SR latch constructed from a pair of cross-coupled NOR gates
...
+
+NOR2.C.connect(NOR1.B)  #Connecting output of second NOR with input of first NOR
```
Comment (rajat974) [inline README.md:30]: no connect method !

File: README.md [hunk 4]
```diff
@@ -21,18 +21,32 @@ Here's an example of SR latch constructed from a pair of cross-coupled NOR gates
 ![SR latch | Source: Wikipedia](https://upload.wikimedia.org/wikipedia/commons/c/c6/R-S_mk2.gif)
 
 ```python
+
 from BinPy import *
 
-a = Connector()
```
Comment (rajat974) [inline README.md:26]: It was written there to show connector support with the gates

**Comment by nikhilkumar13** [comment]
So what is the final example Rajat , right now i am just going through the code

**Comment by rajat974** [comment]
Final Readme is updated one with the develop branch of BinPy codebase. So for that you dont need to worry

**Comment by jayrambhia** [comment]
so what's the issue here? can someone brief me about it?

**Comment by mayureshwaykole** [comment]
The README section referring to SR latch construction is inconsistent with current codebase.

**Comment by debugger22** [comment]
@Mnw2212 Yea it needs to be rewritten, that example was written according to old codebase.

**Pull request closed without merging**